cc/toolchains/llvm/unix_cc_toolchain_config.bzl

```diff
@@ -1300,12 +1331,57 @@ def _impl(ctx):
         ],
     )
 
-    is_linux = ctx.attr.target_libc != "macosx"
     libtool_feature = feature(
         name = "libtool",
         enabled = ctx.attr.use_libtool,
     )
 
+    swift_stdlib_feature = feature(
+        name = "stdlib",
+        enabled = True,
+        # The default is libstdc++, with the option to use libc++ instead.
+        flag_sets = [
+            flag_set(
+                actions = all_cpp_compile_actions + [ACTION_NAMES.lto_backend],
+                flag_groups = ([
+                    flag_group(
+                        flags = [
+                            "-stdlib=libstdc++",
+                        ],
+                    ),
+                ]),
+                with_features = [with_feature_set(not_features = ["libcpp"])],
+            ),
+            flag_set(
+                actions = all_link_actions + lto_index_actions,
+                flag_groups = ([
+                    flag_group(
+                        flags = [
+                            # link statically for now
```

Comment:
is the static linking a change in behavior?

Reply:
These are the same build flags we've been using - I left that comment here because whether to link the standard library statically or dynamically should be a toolchain feature - but it's going to be a bit of work to make that possible.